PermissionScope/PermissionScope.swift

```diff
@@ -201,10 +245,10 @@ import CoreMotion
         closeButton.frame.offsetInPlace(dx: -contentView.frame.origin.x, dy: -contentView.frame.origin.y)
         closeButton.frame.offsetInPlace(dx: 105, dy: -((dialogHeight/2)-20))
         closeButton.frame.offsetInPlace(dx: self.closeOffset.width, dy: self.closeOffset.height)
-        if closeButton.imageView?.image != nil {
-            closeButton.setTitle("", forState: UIControlState.Normal)
+        if let _ = closeButton.imageView?.image {
```

Comment:
Just cleaner, imo at least. _ tells the compiler to ignore the result iirc, so it isn't stored.